README.wine.md

```diff
@@ -1,40 +1,98 @@
+64-bit wine is under somewhat active development so a recent installation is
+recommended. Fedora keeps it up-to-date and ships version 3.15 of both
+32-bit and 64-bit as of the time of writing. This also allow `WOW64` i.e.
+runnning both 32-bit and 64-bit window binaries in the same setting.
+
 For the development of HarfBuzz, the Microsoft shaping technology, Uniscribe,
 as a widely used and tested shaper is used as more-or-less OpenType reference
 implemenetation and that specially is important where OpenType specification
 is or wasn't that clear. For having access to Uniscribe on Linux/macOS these
 steps are recommended:
 
-1. Install Wine from your favorite package manager.
+1. Install Wine and mingw32/mingw64 from your package manager. This likely
+   means, at the very least, running this:
+
+```
+sudo dnf install -y wine.i686 wine.x86_64 mingw32-gcc-c++ mingw64-gcc-c++
+```
+
+2. set up autoconf with `NOCONFIGURE=1 ./autogen.sh`
+
+3. Build the win32 version of harbuzz with:
+
+```
+mkdir win32build
+cd win32build && ../mingw32.sh --with-uniscribe && cd ..
+make -Cwin32build
+```
+
+4. Build the win64 version of harbuzz with:
 
-2. And `mingw-w64` compiler.
-   With `brew` on macOS, you can have it like `brew install mingw-w64`
```

Comment:
Covering macOS also would be nice, we sometime need to test it also and on mac all the three shapers (harfbuzz, uniscribe, coretext) can be present at the same time :)